pkg/context/context_test.go

```diff
@@ -25,3 +26,19 @@ func TestContext_EnvironMap(t *testing.T) {
 	got["c"] = "3"
 	assert.Empty(t, c.Getenv("c"), "Expected to get a copy of the context's environment variables")
 }
+
+func TestContext_SetSensitiveValues(t *testing.T) {
```

Comment:
To test this change, we want to do make a test context like you have done below, but then we should try two cases: setting some sensitive values and then writing to the context's Out and Err fields. Based on the value of ShowSensitiveValues, the captured output should either display *** in place of the sensitive values set, or print them without alteration.